src/foam/parse/EnumParserFactory.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.parse;

```

Comment:
Does this class differ at all from the EnumParserFactory in foam/lib/json/EnumParserFactory.java